flutter/lib/src/navigation/sentry_navigator_observer.dart

```diff
@@ -139,7 +173,6 @@ class SentryNavigatorObserver extends RouteObserver<PageRoute<dynamic>> {
     );
 
     _finishTransaction();
-    _startTransaction(previousRoute);
```

Comment:
i guess it's fine to remove the transaction in didPop, but we should highlight it in the changelog as a "breaking change" (or as a behaviour fix)